lib/database/sthDatabase.js

```diff
@@ -375,7 +376,11 @@ function save2File(attrName, stream, fileName, callback) {
  * @param {function} callback The callback
  */
 function generateCSV(attrName, stream, callback) {
-    var fileName = attrName + '-' + Date.now() + '.csv';
+    if (count === Number.MAX_SAFE_INTEGER) {
+        count = 0;
+    }
+    var fileName = attrName + '-' + Date.now() + '-' + count + '.csv';
```

Comment:
Just to know:

Is this temporal file deleted after use? To avoid having a growing number of file as the STH is used that can starve at the end the resources of the host where STH is running....

Reply:
Yes, this temporary file is deleted after use. The code for same is written [here](https://github.com/telefonicaid/fiware-sth-comet/blob/master/lib/server/handlers/sthGetDataHandler.js#L141).

Reply:
NTC